tools-v2/README.md

```diff
@@ -1421,4 +1800,4 @@ Output:
 | curve_ops_tool list-may-broken-vol   |                                |
 | curve_ops_tool set-copyset-availflag |                                |
 | curve_ops_tool rapid-leader-schedule |                                |
-| curve_ops_tool scan-status           |                                |
+| curve_ops_tool scan-status           | curve bs list/query scan-status|
```

Comment:
add to the bottom which has content.(table entry is continuous)

Reply:
> add to the bottom which has content.(table entry is continuous)

done